### PR TITLE
Adds an explanation of how we can go about testing

### LINKED ISSUE #5: deving with a b-lib component within a Project
- [x] document proposed process for deving with the b-lib within a Project

### PR DESCRIPTION
**Pull request by robguy21**
## Proposed changes
- use  npm built-in feature as described here: https://yarnpkg.com/lang/en/docs/cli/link/

## Types of changes

What type of changes does your code introduce?
_Put an `x` in the boxes that apply, leave blank if none_

- [ ] Frontend
- [ ] Backend
- [ ] Bugfix
- [ ] New feature
- [ ] Component of Major Release
- [ ] Project Management
- [x] Quality of Life

## Relevant Issues

_List Issues below in format `#<ISSUE-NUMBER> [Full|Partial]`_
- Resolves #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
-```bash
-git remote add origin git@github.com:YourUsername/YourRepo.git
 ```
+import { SimplePopup } from 'b-lib';
+```
+
+## When Done with Dev on your Project
 
-Also make sure to update the package.json with your info
+1. install module like so: `yarn add https://github.com/youruser/b-lib#yourtag --force`
```
Comment (sscholle) [inline README.md]: we wont be using personal tags, only branch names

tags will be pushed to master on release

Comment (sscholle) [reply README.md]: @robguy21 can you amend the Deploy process info hre?

**Comment by pruhesi** [comment]
Could we get the set up process and commands for running the tests added to the readme as well?

**Comment by sscholle** [comment]
abandoned this PR due to it's link to an old/private repo, see: #17

**Pull request closed without merging**